catalog/intro.md

```diff
@@ -0,0 +1,13 @@
+Minimal UI Kit for CARTO solutions, with reusable components.
+
+## Installation
+Airship is available as an [npm package](https://www.npmjs.com/package/@carto/airship).
```

Comment:
How about saying "two npm packages: `@carto/airship-styles` and `@carto/airship-components`?